### DISCUSSION TITLE
Arm and Disarm - have to press multiple times

### DISCUSSION THREAD

**Discussion by alkalinitysa** [rainepretorius/olarm-ha-integration#100] [Q&A]
Hi Guys,

I'm a newbie to HomeAssistant, but I'm learning every day :-)
Olarm integration seems to be working great!

The only thing I cant figure out is why the arm/disarm calls are so flaky.

Arm - sometimes happens on the first try, sometimes I have to press it 2-3 times
Disarm - sometimes happens on the first try, sometimes I have to press it 3-4 times

Am I doing something wrong, or is this a known bug?

**Comment by rainepretorius** [comment]
It is a known bug that cannot be resolved without Olarm removing their API limiter or them creating their own integration.
